artifact.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh -e
 
-wget -q -O artifact_node.sh 'https://raw.githubusercontent.com/mdblp/tools/dbl/artifact/artifact_node.sh'
+wget -q -O artifact_node.sh 'https://raw.githubusercontent.com/mdblp/tools/dblp/artifact/artifact_node.sh'
 chmod +x artifact_node.sh
 
 . ./version.sh
-./artifact_node.sh
+./artifact_node.sh node
```

Comment:
Is the "node" parameter usefull for the script artifact_node.sh ?
After viewing this script, it don't seems to have parameter.